src/content/docs/examples/releases-content-layer.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: List recent releases with the Content Layer API
+description: A custom page that lists recent releases for a GitHub repository using the Content Layer API
+---
+
+import StackBlitz from "../../../components/StackBlitz.astro";
+
+This example demonstrates a custom page, which uses the experimental [Content Layer API](https://astro.build/blog/future-of-astro-content-layer/) to list recent releases for a GitHub repository.
```

Comment:
I’m also a bit tempted to avoid the link and “experimental” so we don’t need to update this page in a month or whenever 5.0 is stable. What do you think? There are pros and cons I guess.

```suggestion
This example demonstrates a custom page, which uses Astro’s Content Layer API to list recent releases for a GitHub repository.
```

Reply:
Considering I mention multiple times it's experimental and link to the announcement in my guide, and that now there is a link to it, I think it's fine to remove them. I also have a todo to remove the experimental mention and link to the official docs in my guide once it's released.